views/users.handlebars

```diff
@@ -0,0 +1,15 @@
+<div class="home">
+  <h4 class="sub-heading">Meet our Users</h4>
+  <ol class="user-list">
+{{#each users as |user|}}
+    <li>
+      <article class="users">
+        <div class="user-card">
+            <span class="list-item"><strong>Name:</span>
+          <h4>{{user.first_name}} {{user.last_name}}</h4>
+          <span class="list-item"><strong>Email:</span>
+          <h4>{{user.email}}</h4>
+{{/each}}
```

Comment:
do we want to display post title(s) with each user?

Reply:
I was thinking we could have a page to display all users (first name, last name & email) then a specific user page that displays all of their posts and comments.